src/lib/definition-parser.ts

```diff
@@ -114,7 +115,18 @@ export async function getTypingInfo(folderName: string): Promise<TypingParseFail
 	}
 
 	const hasPackageJson = await fsp.exists(path.join(directory, "package.json"));
-	const allFiles = hasPackageJson ? mi.declFiles.concat(["package.json"]) : mi.declFiles;
+	const allFiles = mi.declFiles.slice();
+	if (allEntryPointFilenames.length > 1) {
+		log.info("Found more than one entry point:")
+		for (let i = 1; i < allEntryPointFilenames.length; i++) {
+			log.info(`Processing additional entry point: ${allEntryPointFilenames[i]}`);
+			const mi = await getModuleInfo(directory, allEntryPointFilenames[i], log.info);
+			allFiles.concat(mi.declFiles);

```

Comment:
We should probably just let `getModuleInfo` take multiple entry points; that way every file is included in the calculation of `referencedLibraries` and friends and we don't have to worry about merging multiple ModuleInfo objects together.